indexeddb-files/tabs/indexeddb-filehandle.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Create temporary files</title>
+    <meta charset="utf-8">
+    <style>
+     .overflow-scroll {
+       max-height: 200px;
+       overflow: auto;
+     }
+    </style>
+  </head>
+  <body>
+    <h1>Create mutable files using the MutableFile API, and then save and retrieve them using IndexedDB API</h1>
+
+    <hr/>
+
+    <h1>1. Create temporary files</h1>
+
+    <div>
+      <button id="create-file">Create a MutableFile</button>
+      <button id="get-metadata">Get Metadata of the opened MutableFile</button>
+    </div>
+
+    <hr/>
+
+    <div>
+      <textarea id="file-data" placeholder="data to save into the file..."
+                style="width: 100%;"></textarea>
+      <button id="save-data">Save Data into the opened MutableFile</button><br/>
```

Comment:
"Save Data into the opened MutableFile" "save" seems strange to me when it hasn't been saved. Could we say "Add data to the MutableFile"?